spec/support/vcr.rb

```diff
@@ -35,6 +35,12 @@
   config.before_record do |i|
     i.response.body.force_encoding('UTF-8')
   end
+  config.filter_sensitive_data('<ACCESS_TOKEN>') do
+    ENV.fetch('ONE_DRIVE_TEST_OAUTH_CLIENT_ACCESS_TOKEN', 'MISSING_ONE_DRIVE_TEST_OAUTH_CLIENT_ACCESS_TOKEN')
+  end
+  config.filter_sensitive_data('<ACCESS_TOKEN>') do
+    ENV.fetch('NEXTCLOUD_LOCAL_OAUTH_CLIENT_ACCESS_TOKEN', 'MISSING_NEXTCLOUD_LOCAL_OAUTH_CLIENT_ACCESS_TOKEN')
+  end
```

Comment:
Nice one, I would add these filter in `modules/storages/spec/spec_helper.rb` - as we can extend module specific VCR configuration there.

E.g. cassette directory is configured per module: https://github.com/opf/openproject/blob/dev/modules/storages/spec/spec_helper.rb#L41

Reply:
good hint, will move